app/js/lib/foxbox/foxbox.js

```diff
@@ -537,7 +369,7 @@ export default class Foxbox extends Service {
    */
   setServiceState(id, state) {
     return new Promise((resolve, reject) => {
-      fetchJSON(`${this.origin}/services/${id}/state`, 'PUT', state)
+      this[p.net].fetchJSON(`${this[p.net].origin}/services/${id}/state`, 'PUT', state)
```

Comment:
I agree.

In the future, I want `this[p.net].fetchJSON` et al. to be clever enough to detect if the URL passed is absolute or relative. If it's relative then it can automatically prepend `${this[p.net].origin}/api/v${this[p.settings].apiVersion}/` so that we can focus more on the calls to the REST API rather than on those verbose details.